docs/doc_sources/api_reference/index.rst

```diff
@@ -0,0 +1,34 @@
+.. _api_reference:
+
+=============
+API Reference
+=============
+
+The package ``dpctl`` provides
+
+* Python API
+   - :ref:`API objects <dpctl_pyapi>` in :py:mod:`dpctl` namespace
+   - :ref:`API objects <dpctl_tensor_pyapi>` in :py:mod:`dpctl.tensor` namespace
```

Comment:
I feel tensor should be a separate bullet point.
`SYCL-based Python Array API library`